lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1232,6 +1233,12 @@ getWalletBalance wallet pending = ApiT $ WalletBalance
     , reward = Quantity 0
     }
 
+getByronWalletBalance :: Wallet s -> Set Tx -> ApiByronWalletBalance
+getByronWalletBalance wallet pending = ApiByronWalletBalance
+    { available = Quantity $ availableBalance pending wallet
+    , total = Quantity $ totalBalance pending wallet
```

Comment:
Not sure if we have to update swagger.yaml.

Reply:
It does need to be updated yes: https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/getByronWallet

Reply:
f8c63cc